save-format.md

```diff
@@ -599,6 +747,15 @@ Tracks whether you ever viewed the player profile. Used to determine if it shoul
 290e: time - Time
 ```
 
+## Player - Character Selection
+
+```
+2a7a: byte - Player 1
```

Comment:
Is this 0x00 for ana through 0x13 for classic guy?

Reply:
Yes